### ISSUE TITLE
Given a wide table, the navigation gets overlaid on the table

### ISSUE THREAD

**Issue by melindafekete** [commercetools/commercetools-docs-kit#1477]
Given a wide table, the navigation gets overlaid on the table (see screenshot below). 

[Preview](https://docs-proxy-cut-124-associate-roles.commercetools.vercel.app/api/associate-endpoints-overview)

![Screenshot 2023-01-11 at 11 49 14](https://user-images.githubusercontent.com/14042876/211787671-150d0249-3181-47be-a732-45f505f8c821.png)

**Comment by emmenko** [comment]
Thanks for reporting.

The main issue is due to the `code` text not having word breaks.

This is how it would look with breaking up the words:

<img width="880" alt="image" src="https://user-images.githubusercontent.com/1110551/211789983-d7065063-7256-4ea8-88e8-d570197478dc.png">

We never really addressed this issue. Breaking up the words is one option. Another option might be to have horizontal scrolling (but we decided against it back in the days if I remember correctly).

What would be your expectation?

**Comment by melindafekete** [comment]
Thanks @nikola-mladenovic For my case, I want to get rid of this table altogether 😅 but just wanted to report the behavior in case it is not yet known. I thought of horizontal scrolling but did not want to jump to any conclusions there. Perhaps Zoe (our UX designer-to-be) can tackle this question in the future. Good to know that the linebreaks would help in the meantime. Feel free to close / keep-open, as you see fit.

**Comment by nkuehn** [comment]
Having wide tables with many columns is never really a good idea in a responsive website that caters to tablets and small screens, too.   It's not really a thing a designer can fix (the screen of the user is only that wide and string literals (inline code) are intentionally not word breaking because this could lead to really weird user issues when copying / understanding them. 

A good practical pattern for tables like this is try to merge information into columns and have them readable enough inside a column.   For example, here I would merge the first two columns and consider a shorter column title for "specification",  especially because it's empty anyways in that table.  

Manual word break tags are possible, too.  You can insert  `<wbr />` at any place to allow the browser to break at that place even if normally not intended to do so.    Maybe it even works inside a markdown inline code. 

TL:DR:  I think this is more fundamental and should be considered a content problem, not a UI or technical one.

**Comment by melindafekete** [comment]
@nkuehn As for this particular case being a content problem, I agree - as I wrote in my comment above, I am removing this table altogether.

I raised the issue just in case it is not known. If it's not something we consider a problem, then let's close it.
